osu.Game.Tests/Visual/UserInterface/TestSceneModSettings.cs

```diff
@@ -53,6 +55,16 @@ public void TestButtonShowsOnModAlreadyAdded()
             AddAssert("button enabled", () => modSelect.CustomiseButton.Enabled.Value);
         }
 
+        [Test]
+        public void TestCustomisationOpensOnModSelect()
+        {
+            createModSelect();
+
+            AddStep("open", () => modSelect.Show());
```

Comment:
Should we have to check that customization is not opened before the selection?

Reply:
OK, I'll do that